### PR TITLE
Update SKILL.tsv

### PR DESCRIPTION
**Pull request by Sunaries**
A continuation of my cleaning of the previous PR https://github.com/Treeofsavior/EnglishTranslation/pull/927. **More to follow soon**. 
I have also edited Rodelero's skill "Evade and move" to "Slithering".

Here is an image on how the updated skill descriptions look like.

GitHub Logo

### CODE REVIEW

File: SKILL.tsv [hunk 1]
```diff
@@ -2587,457 +2587,457 @@ SKILL_20150317_002586	$Increases the additional damage of [Steady Aim] per attri
...
-SKILL_20150317_002682	$Enhanced the damage of [Ice Blast] to Cryomancer's 3rd Circle.
+SKILL_20150317_002682	$Enhanced the damage of [Ice Blast] to Cryomancer's 3rd Circle standards.
 SKILL_20150317_002683	Shield Lob : Enhance
 SKILL_20150317_002684	$Increases the damage of [Shield Lob] per attribute level.
 SKILL_20150317_002685	Rim Blow : Enhance 2
-SKILL_20150317_002686	$Enhanced the damage of [Rim Blow] to Peltasta's 2nd Circle.
+SKILL_20150317_002686	$Enhanced the damage of [Rim Blow] to Peltasta's 2nd Circle standards.
 SKILL_20150317_002687	Rim Blow : Enhance 3
-SKILL_20150317_002688	$Enhanced the damage of [Rim Blow] to Peltasta's 3rd Circle.
-SKILL_20150317_002689	Umbo Blow : Enhance 2
```
Comment (Sunaries) [inline SKILL.tsv]: Check

File: SKILL.tsv [hunk 2]
```diff
@@ -2587,457 +2587,457 @@ SKILL_20150317_002586	$Increases the additional damage of [Steady Aim] per attri
...
 SKILL_20150317_002709	Wagon Wheel : Enhance 3
-SKILL_20150317_002710	$Enhanced the damage of [Wagon Wheel] is enhanced to Highlander's 3rd Circle.
+SKILL_20150317_002710	$Enhanced the damage of [Wagon Wheel] is enhanced to Highlander's 3rd Circle standards.
 SKILL_20150317_002711	Cross Guard : Enhance 2
-SKILL_20150317_002712	$Enhanced the defense increase effect of [Cross Guard] to Highlander's 2nd Circle.
+SKILL_20150317_002712	$Enhanced the defense increase effect of [Cross Guard] to Highlander's 2nd Circle standards.
 SKILL_20150317_002713	Cross Guard : Enhance 3
-SKILL_20150317_002714	$Enhanced the defense increase effect of [Cross Guard] to Highlander's 3rd Circle.
+SKILL_20150317_002714	$Enhanced the defense increase effect of [Cross Guard] to Highlander's 3rd Circle standards.
 SKILL_20150317_002715	Moulient : Enhance 2
```
Comment (Sunaries) [inline SKILL.tsv]: Check

File: SKILL.tsv [hunk 3]
```diff
@@ -2587,457 +2587,457 @@ SKILL_20150317_002586	$Increases the additional damage of [Steady Aim] per attri
...
 SKILL_20150317_002764	Blessing : Enhance
 SKILL_20150317_002765	$Increases the additional damage effect of [Blessing] per attribute level.
 SKILL_20150317_002766	Aspersion : Enhance 2
-SKILL_20150317_002767	$Enhanced the damage of [Aspersion] to Priest's 2nd Circle.
+SKILL_20150317_002767	$Enhanced the damage of [Aspersion] to Priest's 2nd Circle standards.
 SKILL_20150317_002768	Aspersion : Enhance 3
-SKILL_20150317_002769	$Enhanced the damage of [Aspersion] to Priest 3rd Circle.
+SKILL_20150317_002769	$Enhanced the damage of [Aspersion] to Priest 3rd Circle standards.
 SKILL_20150317_002770	Sacrament : Enhance_
-SKILL_20150317_002771	$Enhanced the Holy property attack increase effect of [Sacrament] to Priest's 2nd Circle.
```
Comment (Sunaries) [inline SKILL.tsv]: Check

File: SKILL.tsv [hunk 4]
```diff
@@ -2587,457 +2587,457 @@ SKILL_20150317_002586	$Increases the additional damage of [Steady Aim] per attri
...
 SKILL_20150317_002828	Effigy : Enhance 3
-SKILL_20150317_002829	$Enhanced the damage of [Effigy] to Bokor's 3rd Circle.
+SKILL_20150317_002829	$Enhanced the damage of [Effigy] to Bokor's 3rd Circle standards.
 SKILL_20150317_002830	Damballa : Enhance 2
-SKILL_20150317_002831	$Enhanced the damage of [Damballa] to Bokor's 2nd Circle.
+SKILL_20150317_002831	$Enhanced the damage of [Damballa] to Bokor's 2nd Circle standards.
 SKILL_20150317_002832	Damballa : Enhance 3
-SKILL_20150317_002833	$Enhanced the damage of [Damballa] to Bokor's 3rd Circle.
+SKILL_20150317_002833	$Enhanced the damage of [Damballa] to Bokor's 3rd Circle standards.
 SKILL_20150317_002834	$Mackangdal: Cumulative damage decrease
```
Comment (Sunaries) [inline SKILL.tsv]: Check (uppercase)

File: SKILL.tsv [hunk 5]
```diff
@@ -2587,457 +2587,457 @@ SKILL_20150317_002586	$Increases the additional damage of [Steady Aim] per attri
...
 SKILL_20150317_002874	Impaler : Enhance 2
-SKILL_20150317_002875	[Impaler] damage is enhanced to food Tafurakuto Circle 2.
+SKILL_20150317_002875	$Enhanced the damage of [Impaler] to Cataphract's 2nd Circle standards.
 SKILL_20150317_002876	Impaler : Enhance 3
-SKILL_20150317_002877	[Impaler] damage is enhanced to food Tafurakuto Circle 3.
+SKILL_20150317_002877	$Enhanced the damage of [Impaler] to Cataphract's 3rd Circle standards.
 SKILL_20150317_002878	Meteor : Enhance
-SKILL_20150317_002879	Damage of [Meteor] increases per attribute level
+SKILL_20150317_002879	$Increases the damage of [Meteor] per attribute level.
 SKILL_20150317_002880	Meteor : Enhance_
```
Comment (Sunaries) [inline SKILL.tsv]: Check

File: SKILL.tsv [hunk 6]
```diff
@@ -2587,457 +2587,457 @@ SKILL_20150317_002586	$Increases the additional damage of [Steady Aim] per attri
...
+SKILL_20150317_002910	$Arrow Sprinkle: Enhance 3
+SKILL_20150317_002911	$Enhanced the damage of [Arrow Sprinkle] to Ranger's 3rd Circle standards.
+SKILL_20150317_002912	$Critical Shot: Enhance
+SKILL_20150317_002913	$Increases the damage of [Critical Shot] per attribute level.
 SKILL_20150317_002914	Critical Shot: Enhance 2
-SKILL_20150317_002915	[Critical shot] damage is enhanced to Ranger Circle 2.
-SKILL_20150317_002916	Critical shot: Enhance 3
-SKILL_20150317_002917	[Critical shot] damage is enhanced to Ranger Circle 3.
+SKILL_20150317_002915	$Enhanced the damage of [Critical Shot] to Ranger's 2nd Circle standards.
+SKILL_20150317_002916	$Critical shot: Enhance 3
```
Comment (Sunaries) [inline SKILL.tsv]: Check

File: SKILL.tsv [hunk 7]
```diff
@@ -4825,16 +4825,16 @@ SKILL_20150717_004824	The range that will be applied by [Safety Zone] will be in
...
-SKILL_20150717_004832	Whenever an enemy gets shot by [Effigy], it could be inflicted with [Blind] with 5% probability per Attribute level.
-SKILL_20150717_004833	The range of [Samediveve] will be increased.
-SKILL_20150717_004834	The range of [Ogounveve] will be increased.
-SKILL_20150717_004835	Cooldown time of [Carve Vakarine] will be reduced.
-SKILL_20150717_004836	Out of Body: Avoidance
-SKILL_20150717_004837	While [Out of Body] is being activated, the body's evasion will increase by 10% per Attribute level.
+SKILL_20150717_004828	$Increases the range of [Melstis] to 50. 
+SKILL_20150717_004829	$Sacrament: Dark Property Resistance
+SKILL_20150717_004830	$Increases your Dark property resistance by 5 per attribute level while [Sacrament] is active.
+SKILL_20150717_004831	$Increases the duration of [Revive] by 7 seconds per attribute level.
```
Comment (Sunaries) [inline SKILL.tsv]: Check

File: SKILL.tsv [hunk 8]
```diff
@@ -2587,457 +2587,457 @@ SKILL_20150317_002586	$Increases the additional damage of [Steady Aim] per attri
...
+SKILL_20150317_002915	$Enhanced the damage of [Critical Shot] to Ranger's 2nd Circle standards.
+SKILL_20150317_002916	$Critical shot: Enhance 3
+SKILL_20150317_002917	$Enhanced the damage of [Critical Shot] to Ranger's 3rd Circle standards.
 SKILL_20150317_002918	Time Bomb Arrow: Enhance
-SKILL_20150317_002919	Damage of [Time bomb arrow] increases per attribute level
+SKILL_20150317_002919	$Increases the damage of [Time Bomb Arrow] per attribute level.
 SKILL_20150317_002920	Time Bomb Arrow: Enhance 2
-SKILL_20150317_002921	Damage of the Time Spring Arrow] is enhanced to Ranger Circle 3.
+SKILL_20150317_002921	$Enhanced the damage of [Time Spring Arrow] to Ranger's 3rd Circle standards.
 SKILL_20150317_002922	Ballista: Damage
```
Comment (Sunaries) [inline SKILL.tsv]: Check

File: SKILL.tsv [hunk 9]
```diff
@@ -1005,7 +1005,7 @@ SKILL_20150317_001004	$Frenzy
 SKILL_20150317_001005	$Increases your attack when you use consecutive basic attacks on the target.
 SKILL_20150317_001006	$Attack +#{CaptionRatio}# {nl} Stacks up to #{CaptionRatio2}# times {nl} Duration #{CaptionTime}# seconds
 SKILL_20150317_001007	Aggressor
-SKILL_20150317_001008	$Your attack is guaranteed to hit, but your Critical Rate and Critical Evasion decreases
+SKILL_20150317_001008	$Your attacks are guaranteed to hit, but your Critical Rate and Critical Evasion decreases.
```
Comment (zhouyu47) [inline SKILL.tsv]: decreases -> increases ?

Comment (Sunaries) [inline SKILL.tsv]: It says decreases in the Korean description.

File: SKILL.tsv [hunk 10]
```diff
@@ -3092,22 +3092,22 @@ SKILL_20150323_003091	Evasion rate decrease
 SKILL_20150323_003092	Evasion rate increase
 SKILL_20150323_003093	Equip virtual weapon
 SKILL_20150323_003094	Repair Armor
-SKILL_20150323_003095	Companions is growling. Monster nearby may fall under Fear.
+SKILL_20150323_003095	$Your Companion growls fearfully. Nearby monsters may fall under Fear.
```
Comment (terrapin47) [inline SKILL.tsv:3095]: fearfully -> intimidatingly ?

File: SKILL.tsv [hunk 11]
```diff
@@ -2584,460 +2584,460 @@ SKILL_20150317_002583	Kneeling Shot: Enhance
...
+SKILL_20150317_003034	$Use of Rods
 SKILL_20150317_003035	Can use [Rod] type weapons.
-SKILL_20150317_003036	Use Staff
+SKILL_20150317_003036	$Use of Staves
 SKILL_20150317_003037	Can use [Staff] type weapons.
-SKILL_20150317_003038	Use Mace
+SKILL_20150317_003038	$Use of Maces
 SKILL_20150317_003039	Can use [Mace] type weapons.
-SKILL_20150317_003040	Use wand
+SKILL_20150317_003040	$Use of Wands
```
Comment (grillo-delmal) [inline SKILL.tsv]: [3022-3040] The "of" seem off in those corrections (pun intended)
I liked better the other translation, but how about changing it to "Enable [Weapon type]", and on the description "Enables to use [Weapon type] type weapons."?

**Comment by Sunaries** [comment]
Taking @Grillo-del-mal's suggestion into account, I changed the description a little bit. However, now the differences in this file cannot be changed due to new additions and deletions. I tried changing the file 12 times, removing some "$" marks in the hope that the differences in this file becomes viewable. 

But, no it didn't work. To the point that it frustrates me... - @imcgames Even though I am oppossed to files where I can't see the differences because of the large amount of additions and deletions, I can assure you that this pull request is a positive change for SKILL. This is just only a part of what I'm submitting. I've contributed a lot to the Github repository, so I hope you understand this.

**Comment by grillo-delmal** [comment]
This is why they recomend to maintain pull requests no longer than 150 changes... you can still check the changes per commit without problems, i'll check 'em in a couple of secs

**Comment by Sunaries** [comment]
> This is why they recomend to maintain pull requests no longer than 150 changes... you can still check the changes per commit without problems, i'll check 'em in a couple of secs

@Grillo-del-mal Once I'm in a session where I want to fix the mistranslations and erroneous descriptions, I forget to keep track on how much I've edited because of. This file requires a high priority to avoid confusion with skills and attributes in the next CBT for the players.

Good to merge.

**Comment by imcgames** [comment]
Thank you :smile:

**Comment by ttgmichael** [comment]
I actually can confirm with Notepad that the changes made in this PR is very similar to the the overwritten PR #... nvm xD

**Pull request merged**